src/extensions/WorkspaceView/WorkspaceRecordingView.tsx

```diff
@@ -39,18 +39,25 @@ const WorkspaceRecordingView: FunctionComponent<Props> = ({ recording, sortings,
     }
   }, [selection, recordingInfo])
 
+  const handleBackToRecordings = useCallback(() => {
+      workspaceRouteDispatch({
+        type: 'gotoRecordingsPage'
+      })
+  }, [workspaceRouteDispatch])
+
   if (!recordingInfo) return <div>Loading recording info</div>
 
   return (
     <div style={{margin: 20}}>
       <Grid container spacing={3}>
-        <Grid item xs={12} lg={6}>
+        <Grid item xs={12} xl={6}>
+          <Button onClick={handleBackToRecordings}>Back to recordings</Button>
```

Comment:
You know, I'd always just clicked the 'LABBOX-EPHYS' label in the purple header. But it's nice to make this explicit, particularly since a user asked for it.

Reply:
This especially targets the jupyter widget where there is no LABBOX-EPHYS button.